### ISSUE TITLE
Order Tracker

### ISSUE THREAD

**Issue by sean8300** [ListerAssister/issues#57]
Have an UNRESTRICTED view of a listing and where we are with all the items that are being done on it.  Something like a pizza tracker but for each service listed.  I.E., photos are complete and being edited, MLS is waiting for input form, drone is edited and uploaded, etc etc.
Would have to be careful about what information to provide since it is in unrestricted, which means anyone anywhere can view it (so no names, numebrs, emails, etc....could get tricky)
Submitted By: Sean

**Comment by sean8300** [comment]
https://github.com/ListerAssister/issues/issues/433 is a dupe, keeping this one open and closing the other

**Comment by sean8300** [comment]
Brain storming here but I think I build this modularly.  

Have the actual information on one controller, it would use a simple twig file that only shows THAT relevant information

On the order tracker page it embeds that twig file instead of the order tracker page being the entire function

Doing it this way will allow me to include that specific portion on the customers listing details on a new tab (without having to build this twice, once for the listing details and once for the tracker page)

**Comment by sean8300** [comment]
Ran into a roadblock on Friday and asked for info in Slack, I should have known to put it in here so I can track lack of response, but forgot to.  So Friday I asked for guidance, asked again this morning

**Comment by sean8300** [comment]
This is going the typical route, think I may ask Juvi to ask questions, he seems to get answers.  For now guess I'll build it out and will have Juvi introduce it to everyone, see if people provide feedback/answers/any acknowledgement at all

**Comment by sean8300** [comment]
NOTES FROM MEETING WITH CHAD:
This will end up being split into parts. 

Order Tracker
MLS Queueing
Order Tracker Rescheduling

**Comment by sean8300** [comment]
Going to turn this into a project and create a new tix for each portion.  Closing this, will be linking just for reference

**Comment by sean8300** [comment]
Perfect example listing:
https://dashboard.listerassister.com/public/order-tracker/Pyxak6

**Comment by sean8300** [comment]
Waiting on feedback, AFAIK this is GTG

**Comment by sean8300** [comment]
We need the order tracker to be able to tell if something is put on hold, in order to do that we need a new column under the appointment table for "hold timestamp" or whatever we want to call it.  Reps will be able to place something on hold, which will add the timestamp there, which will cause the order tracker to then display "On Hold" status (it will have to be after completed ts in display order preference) 

Actually going to need two new database fields
appointment.hold_ts - a rep put an entire appointment on hold
appointment_service.not_done - field employee marked something as NOT complete, this will require a new button on employee side

When an appointment is put on hold once it comes out you will actually have to wipe out the on_hold TS, otherwise you will be created circular logic when figuring out whether to show something as complete or on hold on the order tracker

**Comment by sean8300** [comment]
this was scrapped and we are instead just doing online rescheduling
